conf/messages.cy

```diff
@@ -450,4 +450,45 @@ ukInsuranceGains.none = No, I did not make any UK insurance gains
+ukInsuranceGains.agent.error.required = Select if your client made gains from UK insurance policies or voided ISAs, or ‘No, my client did not make any UK insurance gains’
+
+ukDividendsSharesAndLoansFromLimitedCompanies.title = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.title = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.heading = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.heading = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividends.summary = Understanding dividends
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividends.p1 = You may get a dividend payment if you own shares in a company. Dividends can be cash or ‘stock’ (shares). You can earn a total of £1,000 from UK and non-UK dividends each year without paying tax. You must tell us about all of your dividend income, even if it’s less than £1,000.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.taxOnDividends.linkText = Learn more about tax on dividends (opens in a new tab)
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.summary = Understanding dividends from unit trusts or investment companies
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p1 = Investment trusts make money through buying and selling shares or assets in other companies.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p2 = Unit trusts make money by buying and selling bonds or shares on the stock market. The fund is split into units which investors buy. A fund manager creates and cancels units when investors join and leave the trust.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p3 = Open-ended investment companies are like unit trusts, but create and cancel shares, rather than units, when investors join or leave.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p4.linkText = Find out more about authorised investment funds (opens in a new tab)
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingFreeRedeemableShares.summary = Understanding free or redeemable shares
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingFreeRedeemableShares.p1 = You can hold both free and redeemable shares.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingFreeRedeemableShares.p2 = Redeemable shares are ones a company can buy back at an agreed price on a future date.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingFreeRedeemableShares.p3 = Free additional shares are also known as ''bonus issues of securities''.
```

Comment:
I think this should be: ‘bonus issues of securities’ although would be worth double checking this one.

Reply:
Yeah you're most likely to be right, I've asked Rosie to clarify what this should be andI'll make the amemdment in my next commit, Thanks.

Reply:
@ghousedev I've pushed my latest commit to resolve this query, is this what you were expecting?